plugins/Kaleidoscope-OneShot/src/kaleidoscope/plugin/OneShot.cpp

```diff
@@ -128,18 +129,18 @@ bool OneShot::isStickable(Key key) {
     if (n < oneshot_key_count) {
       return bitRead(stickable_keys_, n);
     }
-#ifndef ONESHOT_WITHOUT_METASTICKY
-  } else if (key == OneShot_MetaStickyKey) {
-    return true;
-#endif
   }
-  return false;
+  return true;
```

Comment:
I'm not sure I understand why this is changed from returning `false` to `true`. Calling `OneShot.isStickableDefault(Key_A)` should return `false` under normal circumstances.

Reply:
I disagree.  There is much more utility in a _sticky_ `A` than a _one-shot_ `A`.

Reply:
By default, there's no setting in OneShot that would allow any non-modifier, non-layer shift key to become sticky, because normally it would need to enter the one-shot state first, so this doesn't change anything for someone using OneShot on its own, with no customizations.

But if the default here is `false`, and the user wants to use the `OneShot_MetaStickyKey`, which itself acts like a OneShot key, that key will be either blocked from becoming sticky, or the user will need to override the `OneShot.isStickable()` function.  We can't just have the OneShotMetaKeys plugin override `OneShot.isStickable()`, because that would prevent the user from overriding it, and/or result in multiple `weak` definitions of the same function, which probably results in undefined behaviour or link-order dependence.

It also means that if someone wants to write a very simple plugin like the `OneShotInsert` plugin in the testcase for 1061, they would also need to override `OneShot.isStickable()` in order to make the chosen key(s) behave like a "normal" OneShot key.

The default should be `true`, which is the same as the default behaviour for `OSM()` & `OSL()` keys—those ones need extra code added to turn off stickiness, too.

Reply:
With this explanation, the change makes sense, thank you! Can we have a little explanation above the return, so whoever reads it in the future, will not be as confused as I was?

Reply:
Done!